### DISCUSSION TITLE
Partial and whitespace control

### DISCUSSION THREAD

**Discussion by LastDragon-ru** [handlebars-lang/handlebars.js#1874] [Q&A]
# Question

I want to remove whitespaces around the partial (that I don't want/cannot edit). I've tried to use `{{~> myPartial~}}` but seems it has no effect :(

Is there any way to render this
![image](https://user-images.githubusercontent.com/1329824/178099527-0f4516d7-f0d0-4e92-aa48-58ffd048d8eb.png)

as

```
My Content!
My Content!
```

?

# Example link

https://handlebarsjs.com/playground.html#format=1&currentExample=%7B%22template%22%3A%22%7B%7B%23*inline%20%5C%22myPartial%5C%22%7D%7D%5Cn%20%20My%20Content%5Cn%20%20%5Cn%7B%7B%2Finline%7D%7D%5Cn%7B%7B%23each%20people%7D%7D%5Cn%20%20%7B%7B~%3E%20myPartial~%7D%7D!%5Cn%7B%7B%2Feach%7D%7D%22%2C%22partials%22%3A%5B%5D%2C%22input%22%3A%22%7B%5Cn%20%20people%3A%20%5B%5Cn%20%20%20%20%7B%20firstname%3A%20%5C%22Nils%5C%22%20%7D%2C%5Cn%20%20%20%20%7B%20firstname%3A%20%5C%22Yehuda%5C%22%20%7D%2C%5Cn%20%20%5D%2C%5Cn%7D%5Cn%22%2C%22output%22%3A%22%20%20My%20Content%5Cn%20%20%5Cn!%5Cn%20%20My%20Content%5Cn%20%20%5Cn!%5Cn%22%2C%22preparationScript%22%3A%22%2F%2F%20Handlebars.registerHelper('loud'%2C%20function(string)%20%7B%5Cn%2F%2F%20%20%20%20return%20string.toUpperCase()%5Cn%2F%2F%20%7D)%3B%5Cn%22%2C%22handlebarsVersion%22%3A%224.7.7%22%7D

**Comment by jaylinski** [comment]
This template would work:
```
{{#*inline "myPartial"}}My Content{{/inline}}
{{#each people}}
  {{~> myPartial~}}!
{{/each}}
```
But I guess you want to format your template so it isn't a single long line?

**Reply by LastDragon-ru** [reply]
[Yep](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-conventionalcommits/templates/commit.hbs). I wanted to edit only [main template](https://github.com/conventional-changelog/conventional-changelog/blob/master/packages/conventional-changelog-conventionalcommits/templates/template.hbs) to add "breaking change" mark right after `{{> commit root=@root}}`. 

(right now I copy-pasted all templates)
